packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -284,215 +222,6 @@ QUnit.test('`if` helper with inline form: can use truthy param as binding', func
   equal(view.$().text(), 'yes');
 });
```

Comment:
I'm happy to drop this. Let me know if you object, otherwise I'll delete it before merging

TODO: 

 * [ ] Either delete this or rewrite it in shared conditionals